compiler/luci-interpreter/pal/cmsisnn/PALSVDF.h

```diff
@@ -30,8 +30,8 @@ IntegerSVDF(const TfLiteSVDFParams &params, const tflite::RuntimeShape &input_sh
             const int16_t *weight_time_data, const tflite::RuntimeShape &bias_shape,
             const int32_t *bias_data, int16_t *activation_state_data,
             const tflite::RuntimeShape &output_shape, int8_t *output_data, int32_t *scratchpad_data,
-            int32_t *output_temp_data, int32_t scale_1_a, int32_t scale_1_b, int32_t scale_2_a,
-            int32_t scale_2_b, int32_t input_zp, int32_t output_zp)
+            int32_t *output_temp_data, int32_t scale_1_a, int scale_1_b, int32_t scale_2_a,
+            int scale_2_b, int32_t input_zp, int32_t output_zp)
```

Comment:
> types in SVDF kernel: int32_t to int.

Q) can you please share why we need this change? `int` will be different in 32 vs 64 bit architecture.

Reply:
The signature of tflite::reference_ops::EvalIntegerSVDF:
```C++
inline void EvalIntegerSVDF(
    const TfLiteSVDFParams* params, const RuntimeShape& input_shape,
    const int8_t* input_data, const RuntimeShape& weights_feature_shape,
    const int8_t* weights_feature_data, const RuntimeShape& weights_time_shape,
    const int16_t* weights_time_data, const RuntimeShape& bias_shape,
    const int32_t* bias_data, int16_t* state_data,
    const RuntimeShape& output_shape, int8_t* output_data,
    int32_t* scratch_data, int32_t* output_temp_data, int32_t scale_1_a,
    int scale_1_b, int32_t scale_2_a, int scale_2_b, int32_t input_zp,
    int32_t output_zp)
 ```
 so scale_1_b and scale_2_b should be int (causes `error: invalid conversion from 'int32_t* {aka long int*}' to 'int*' [-fpermissive]`
)

Reply:
OK thanks for sharing the info.